external/flume-sink/src/main/scala/org/apache/spark/streaming/flume/sink/Logging.scala

```diff
@@ -26,20 +26,20 @@ import org.slf4j.{Logger, LoggerFactory}
 private[sink] trait Logging {
   // Make the log field transient so that objects with Logging can
   // be serialized and used on another machine
-  @transient private var log_ : Logger = null
```

Comment:
We can't write like `log_: Logger`. So, can we rename those vars to which starts with `_`?